daemon/firewall/firewall_manager_test.go

```diff
@@ -0,0 +1,995 @@
+package firewall
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"net/netip"
+	"strings"
+	"testing"
+
+	"github.com/NordSecurity/nordvpn-linux/daemon/device"
+	"github.com/NordSecurity/nordvpn-linux/meshnet"
+	"github.com/NordSecurity/nordvpn-linux/test/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+var ErrIptablesFailure = errors.New("iptables failure")
+
+type IptablesMock struct {
+	isErr       bool
+	Commands    []string
+	errCommands map[string]bool
+}
+
+func NewIptablesMock(isErr bool) IptablesMock {
```

Comment:
We have to add IptablesMock here, temporarily, against our uts conventions. Adding it in firewall mock directory will cause cyclical dependency, because `nordvpn-linux/test/mock/firewall/firewall.go` depends on the firewall package:
```
// Add and apply firewall rules
func (mf *FirewallMock) Add(rules []firewall.Rule) error {
	mf.Rules = append(mf.Rules, rules...)

	return nil
}
```

This will be fixed once new firewall layer is integrated into networker, because we will be able to remove the old mock.